tests/main.test.ts

```diff
@@ -119,7 +120,9 @@ describe("User start/stop", () => {
     expect(updatedIssue?.assignees).toEqual([{ login: STRINGS.USER, id: 2 }]);
 
     const comments = db.issueComments.getAll();
-    expect(comments[comments.length - 1].body).toEqual(JSON.stringify({ body: `@${STRINGS.USER}, this task has been idle for a while. Please provide an update.` }));
+    const latestComment = comments[comments.length - 1];
+    const partialComment = "@user2, this task has been idle for a while. Please provide an update.\\n\\n\\n<!-- Ubiquity - Followup -"
```

Comment:
Perhaps it makes sense to use the real metadata writing method instead of manually trying to write the metadata.